.vscode/tasks.json

```diff
@@ -0,0 +1,12 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
```

Comment:
JSON does not have comments.

Reply:
... this comes directly from VisualStudio Code.

Reply:
... I thought so. Not blocking.